pkg/sdk/parsers.go

```diff
@@ -52,3 +52,18 @@ func ParseCommaSeparatedSchemaObjectIdentifierArray(value string) ([]SchemaObjec
 	}
 	return ids, nil
 }
+
+// ParseCommaSeparatedAccountIdentifierArray can be used to parse Snowflake output containing a list of account identifiers
+// in the format of ["organization1.account1", "organization2.account2", ...],
+func ParseCommaSeparatedAccountIdentifierArray(value string) ([]AccountIdentifier, error) {
```

Comment:
nit: do we have unit test for this?

Reply:
not yet, i will add them in next PR, as well as tests for the customDiff